src/main/webapp/app/course/competencies/forms/prerequisite/prerequisite-form.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, EventEmitter, Input, OnChanges, OnInit, Output } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import { CompetencyService } from 'app/course/competencies/competency.service';
+import { LectureUnitService } from 'app/lecture/lecture-unit/lecture-unit-management/lectureUnit.service';
+import { CourseCompetencyFormComponent, CourseCompetencyFormData } from 'app/course/competencies/forms/course-competency-form.component';
+import { TranslateService } from '@ngx-translate/core';
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { CommonCourseCompetencyFormComponent } from 'app/course/competencies/forms/common-course-competency-form.component';
+import { CourseCompetencyType } from 'app/entities/competency.model';
+
+@Component({
+    selector: 'jhi-prerequisite-form',
+    templateUrl: './prerequisite-form.component.html',
+    styleUrls: ['./prerequisite-form.component.scss'],
+    standalone: true,
+    imports: [ArtemisSharedModule, CommonCourseCompetencyFormComponent],
+})
+export class PrerequisiteFormComponent extends CourseCompetencyFormComponent implements OnInit, OnChanges {
+    @Input()
+    formData: CourseCompetencyFormData = {
+        id: undefined,
+        title: undefined,
+        description: undefined,
+        softDueDate: undefined,
+        taxonomy: undefined,
+        masteryThreshold: undefined,
+        optional: false,
+        connectedLectureUnits: undefined,
+    };
+
+    @Output()
+    formSubmitted: EventEmitter<CourseCompetencyFormData> = new EventEmitter<CourseCompetencyFormData>();
+
+    readonly CourseCompetencyType = CourseCompetencyType;
+
+    constructor(fb: FormBuilder, lectureUnitService: LectureUnitService, competencyService: CompetencyService, translateService: TranslateService) {
```

Comment:
Should this require a `CompetencyService` or a `PrerequisiteService`? The tests seem to assume that it should be a `PrerequisiteService`, but that doesn't seem quite right.

Changing it to a `PrerequisiteService` fixes another test